Library/Formula/duck.rb

```diff
@@ -2,9 +2,9 @@ class Duck < Formula
   desc "Command-line interface for Cyberduck (a multi-protocol file transfer tool)"
   homepage "https://duck.sh/"
   # check the changelog for the latest stable version: https://cyberduck.io/changelog/
```

Comment:
Yes, I checked Cyberduck.io it only list Cyberduck changes not CLI. And `https://dist.duck.sh/` appears to be a dump from somewhere else. SVN & Git is up to date. 
https://g.iterate.ch/projects/ITERATE/repos/cyberduck/browse